src/tribler/core/utilities/tracker_utils.py

```diff
@@ -116,11 +121,14 @@ def parse_tracker_url(tracker_url):
     if tracker_url.lower().startswith('http://') and ':80/' in tracker_url:
         tracker_url = tracker_url.replace(':80/', '/', 1)
 
+    if tracker_url.lower().startswith('https://') and ':443/' in tracker_url:
+        tracker_url = tracker_url.replace(':443/', '/', 1)
+
     if tracker_url != get_uniformed_tracker_url(tracker_url):
         raise MalformedTrackerURLException(f'Could not sanitize url ({tracker_url}).')
 
     url = urlparse(tracker_url)
-    if not (url.scheme == 'udp' or url.scheme == 'http'):
+    if url.scheme not in ('udp', 'http', 'https'):
```

Comment:
NIT: I believe it is better to use a set here (see the explanation above).

Reply:
Addressed